lib/manageiq/api/client/server.rb

```diff
@@ -0,0 +1,62 @@
+module ManageIQ
+  module API
+    module Client
+      class Server
```

Comment:
As mentioned earlier...I don't understand what Server is different from Client...I think this can just *be* the client, and the extra nesting can go away.